html/chapter7.html

```diff
@@ -1214,7 +1214,7 @@ <h3>THERMAL ENERGY FROM AN AIR BURST</h3>
 
       <aside>
 
-        <p>The curves in Fig. 7.84 show the variation with the scaled time $T/t_{max} of the scaled fireball power, P/P<sub>max</sub></sub> (left ordinate) and of the percent of the total thermal energy emitted, ElE,oI (right ordinate), in the thermal pulse of an air burst.</p>
+        <p>The curves in Fig. 7.84 show the variation with the scaled time $T/t_{max}$ of the scaled fireball power, P/P<sub>max</sub></sub> (left ordinate) and of the percent of the total thermal energy emitted, ElE,oI (right ordinate), in the thermal pulse of an air burst.</p>

```

Comment:
Is there an extra `</sub>` in there in the original, or am I missing an opening `<sub>` somewhere?

Reply:
It does seem hard to see how there is not an extra `</sub>` in the original there, unless there is some strange fixup going on.